api/v1beta1/openstackclient_webhook.go

```diff
@@ -94,7 +94,7 @@ func (r *OpenStackClient) Default() {
 
 }
 
-// +kubebuilder:webhook:verbs=create;update;delete,path=/validate-osp-director-openstack-org-v1beta1-openstackclient,mutating=false,failurePolicy=fail,sideEffects=None,groups=osp-director.openstack.org,resources=openstackclients,versions=v1beta1,name=vopenstackclient.kb.io,admissionReviewVersions={v1,v1beta1}
+// +kubebuilder:webhook:verbs=create;update;delete,path=/validate-osp-director-openstack-org-v1beta1-openstackclient,mutating=false,failurePolicy=fail,sideEffects=None,groups=osp-director.openstack.org,resources=openstackclients,versions=v1beta1,name=vopenstackclient.kb.io,admissionReviewVersions=v1
 
 var _ webhook.Validator = &OpenStackIPSet{}
```

Comment:
Not that this PR added this, but it seems that a lot of these `<x>_webhook.go` files have `var _ webhook.Validator = &OpenStackIPSet{}` instead of the CRD struct that actually pertains to them

Reply:
good catch!! I'll fix that as part of the PR I submitting later today